app/controllers/content_items_controller.rb

```diff
@@ -1,9 +1,17 @@
 require 'gds_api/content_store'
+require 'slimmer/headers'
 
 class ContentItemsController < ApplicationController
+  include Slimmer::Headers
   rescue_from GdsApi::HTTPForbidden, with: :error_403
 
   def show
+    set_slimmer_headers(
+      search_parameters: {
+        "filter_manual" => "/service-manual"
+      }.to_json,
+    )
```

Comment:
:clap: for working out how this works. Could you comment it a bit? It's a bit mind bending even when you know how it works..